content/en/blog/2024/happy-7th-birthday/index.md

```diff
@@ -0,0 +1,200 @@
+across industries," said Chris Aniszczyk, CTO of CNCF. "Istio has helped drive part of this maturation since they
+graduated last year in CNCF and we wish them a fantastic birthday. We look forward to watching and supporting this
+continued growth as the Istio team adds new features like ambient mode and simplifies the service mesh experience."
+
+"Service Meshes are core to microservice architectures, a hallmark of cloud native," said Emily Fox, CNCF TOC
+chair and Senior Principal Software Engineer at Red Hat. "Istio's birthday celebrates the proliferation and
+importance not only of observability and traffic management, but the increasing demand for secure-by-default
+communications through encryption, mutual authentication, and many other core security tenets that simplify the
+adoption, integration, and deployment experience."
+
+"In my opinion Istio isn’t a service mesh. It’s a collaborative community of users and contributors who happen to
+deliver the world’s most popular service mesh," said Mitch Connors, Istio Technical Oversight Committee member and
+Principal Engineer at Microsoft. "Happy birthday to this amazing community! It’s been a fantastic seven years, and
+I’m looking forward to celebrating many more with my friends and colleagues from around the world in the Istio community!"
+
+"It has been a privilege and a fulfilling experience to be part of the world's most popular service mesh team for
+the past two years," Faseela K, Istio Steering Committee Member and Cloud Native Developer at Ericsson. "Happy to
+see Istio grow from a CNCF incubating to graduated project, and even happier to see the momentum and passion with
+which the latest and greatest 1.22 release was done. Wishing many more successful releases in the coming years."
+
+"It has been a privilege to have worked with the Istio community over the last 5 years. There has been an
+abundance of contributors whose dedication, passion, and hard work have made my time on the project truly
+enjoyable," said Eric Van Norman, Istio Technical Oversight Committee member and Advisory Software Engineer at
+IBM. "The community has many users who provide feedback to help make Istio the best service mesh. I continue to be
+amazed by what the community does, and look forward to seeing what successes we will have in the future."
+
+"What makes Istio unique is the community full of developers, users, and vendors from all across the globe working
+together to make Istio the best and most powerful open service mesh in the industry," said Neeraj Poddar, Istio
+Technical Oversight Committee member and VP of Engineering at Solo.io. "It’s the strength of the community that
+has made Istio so successful and now under CNCF I look forward to seeing Istio as the de facto service mesh
+standard for all cloud native applications."
+
+## Continuous Technical Innovation
+
+We are firm believers that diversity drives innovation. What amazes me most is the continuous innovation from the
```

Comment:
What amazes "me"--> is that correct here?

Reply:
corrected